internal/commands/list.go

```diff
@@ -123,9 +124,11 @@ func refreshList(m model) func() tea.Msg {
 
 		m.errors = es
 		// return tea.Batch(m.list.SetItems(convertItems(items)), m.list.NewStatusMessage("Refreshed."))
+		// get date in YYYY-MM-DD hh:mm:ss
+		now := time.Now().Format("2006-01-02 15:04:05")
 		return listUpdate{
 			items:  convertItems(items),
-			status: "Refreshed.",
+			status: fmt.Sprintf("Refreshed at %s.", now),
```

Comment:
As the only method for refresh currently is the `nom refresh` command I don't think a timestamp is needed. 

This can be something to add with the TUI background update and maybe shown somewhere dedicated in the UI (bottom or top right maybe) with a relative date
E.g. Updated 5m ago